modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -82,23 +82,12 @@
             :items="items"
```

Comment:
`AposDocsManagerDisplay` is also emitting `change`, which is ignored. Also `updated`, which can probably be removed.

Reply:
The `emits` array on that component is generally no longer accurate.

Reply:
The change thing is bugging me now, I'll try to get to the bottom of it.

Reply:
OK it's not ignored, there's v-model configuration wiring it up:

```
  model: {
    prop: 'checked',
    event: 'change'
  },
```